src/main/kotlin/no/nav/familie/ef/sak/integration/OppdragClient.kt

```diff
@@ -47,4 +48,9 @@ class OppdragClient(@Value("\${FAMILIE_OPPDRAG_API_URL}")
         return postForEntity<Ressurs<String>>(konsistensavstemmingUriV2, konsistensavstemmingRequest).getDataOrThrow()
     }
 
+    override val pingUri = postOppdragUri
+
+    override fun ping() {
+        operations.optionsForAllow(pingUri)
```

Comment:
Jag förstår att detta är en helsesjekk mot oppdrag, men hur funkar operations och optionsForAllow ?

Reply:
Vad tænker du på med "hur funkar operations"? Operations ær interfacet før `RestTemplate`
`optionsForAllow` ær det samma som att skicka en request som OPTIONS (istællet før GET). Husker ikke hvorfor jeg la til options men tror jeg gjorde det fordi oppdrag ikke hadde en endepunkte for ping og ønsket å sjekke att jeg fikk kontakt
Er kanskje mer riktig å legge til en pingController i oppdrag, eller bruke HEAD.